packages/iTwinUI-react/src/core/Tabs/Tabs.tsx

```diff
@@ -13,6 +13,26 @@ import {
 import '@itwin/itwinui-css/css/tabs.css';
 import { Tab } from './Tab';
 
+export type TabsOrientationProps =
```

Comment:
this is an intermediate type that doesn't need to be exported

Reply:
I saw the `Select.tsx` exported the intermediate type on [L84](https://github.com/iTwin/iTwinUI-react/blob/main/packages/iTwinUI-react/src/core/Select/Select.tsx#L84), so I thought I had to do the same in Tabs.

Removed the export keyword from Tabs.